src/libretro/freej2me_libretro.c

```diff
@@ -605,4 +665,4 @@ bool isRunning(pid_t pid)
 		return true;
 	}
 	return false;
-}
+}
```

Comment:
I'm surprised you didn't get a warning about this one.  From ANSI C 1989 Standard 2.11.2 (2): "A source file that is not empty shall end in a new-line character, which shall not be immediately preceded by a backslash."  The C99 and C11 standards have similar wording, both conveniently in sections 5.1.1.2 (2).  

Here's a text file of the draft:  http://port70.net/~nsz/c/c89/c89-draft.txt

I appreciate the effort on formatting, though probably I should have explained myself a bit better.   I use the terms 'indentation' and 'alignment' to refer to different things.  I use 'indentation' to mean white space at the beginning of a line.  I use 'alignment' to refer to the use of white space in the middle of a line used to align similar parts of adjacent lines.  E.g.  You've tried to align the start of the string literals on lines 73-88 using tabs.  I would use spaces there, as different editors will have different tab stops, which could leave things misaligned.  That's what I meant by using tabs for indentation and spaces for alignment.    Sorry about the confusion.

It looks okay otherwise.

Reply:
Huh, yeah, for some reason my compiler isn't complaining at all about the lack of the newline at the end there... anyway, i'll be pushing a updated commit with the changes requested on this latest comment.

As for the indentation and alignment paragraph: No problem, i think i confused it myself really, since i'm not used to that type of indentation (i mostly use K&R).